src/structures/Invite.js

```diff
@@ -154,7 +145,7 @@ class Invite {
    * @returns {Promise<Invite>}
    */
   delete(reason) {
-    return this.client.api.invites(this.code).delete({ reason }).then(() => this);
```

Comment:
invites and invite go to the same place
https://discordapp.com/api/v7/invite/discord-api
https://discordapp.com/api/v7/invites/discord-api